### ISSUE TITLE
feature: not only for markdown

### ISSUE THREAD

**Issue by youwi** [holgerbrandl/pasteimages#13]
i want paste images save as file ,not only for editing markdown  ..

**Comment by holgerbrandl** [comment]
But what would be the paste format if the target document is not markdown.

In general the plugin could work by adding more document types under
https://github.com/holgerbrandl/pasteimages/blob/master/src/img2md/PasteImageHandler.java#L60

**Comment by youwi** [comment]
thanks for replay.  can config that ?  idea -->menu--Preferences --tools-->config

**Comment by holgerbrandl** [comment]
So what other language would be a usecase? Apart from markdown none has a preview mode in IJ afaik.

Technically such a config panel could be easily implemented I guess.

**Comment by youwi** [comment]
thank you. i am learning.

**Comment by holgerbrandl** [comment]
Makes sense. Would you willing to provide a PR to paste latex code in case the document type is latex?

**Comment by lhotari** [comment]
This would be also useful for AsciiDoc

**Comment by holgerbrandl** [comment]
As I said, the PR seems really straightforward. Feel welcome to contribute.
